.changeset/config.json

```diff
@@ -0,0 +1,10 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@1.5.0/schema.json",
+  "changelog": "@changesets/cli/changelog",
+  "commit": false,
+  "linked": [],
+  "access": "restricted",
```

Comment:
We should add `private: true` to our package.json files (all of them apparently) if we don't want to publish to npmjs

Reply:
https://github.com/atlassian/changesets/blob/master/docs/config-file-options.md#access-restricted--public

Reply:
We should also probably "link" front and back: https://github.com/atlassian/changesets/blob/master/docs/config-file-options.md#linked-array-of-package-names